.github/workflows/ci.yml

```diff
@@ -19,28 +19,69 @@ concurrency:
 
 jobs:
   prepare:
+    name: Check how tests should be executed
     runs-on: ubuntu-20.04
     outputs:
       uuid: ${{ steps.uuid.outputs.value }}
       commit-message: ${{ steps.commit-message.outputs.value }}
+      is-label-present: ${{ steps.label-present.outputs.value }}
+      cypress: ${{ steps.cypress.outputs.value }}
+      synpress: ${{ steps.synpress.outputs.value }}
+
     steps:
       - name: ⬇️ ・Checkout repo
         uses: actions/checkout@v3
         with:
           fetch-depth: '50'
 
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+
       - name: Generate unique ID 💎
         id: uuid
-        run: echo "::set-output name=value::sha-$GITHUB_SHA-time-$(date +"%s")"
+        run: |
+          echo "::set-output name=value::sha-$GITHUB_SHA-time-$(date +"%s")"
+          echo "sha-$GITHUB_SHA-time-$(date +"%s")"
```

Comment:
Why we need this UUID?

Reply:
Cypress dashboard uses commit sha as a way to identify single test run, in order to be able to run tests using rerun button i added timestamp to this sha